packages/reactotron-server/src/app/commands/ApiResponse.tsx

```diff
@@ -1,8 +1,88 @@
 import React from "react"
 import { Command } from "reactotron-core-ui"
 
-export function ApiResponseTimeline({ command }: { command: Command }) {
-  return <span>{JSON.stringify(command.payload.request)}</span>
+interface Props {
+  command: Command
+}
+
+interface State {
+  expanded: null | string
+}
+
+export class ApiResponseTimeline extends React.Component <Props, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      expanded: null,
+    }
+  }
+
+  _renderContentsBlock() {
```

Comment:
I prefer `private renderContentsBlock()` instead of the `_` prefix now that we have a way to represent visibility modifiers in this language.